docs/reference/analysis/tokenfilters/keyword-repeat-tokenfilter.asciidoc

```diff
@@ -1,90 +1,164 @@
 [[analysis-keyword-repeat-tokenfilter]]
-=== Keyword Repeat Token Filter
+=== Keyword repeat token filter
+++++
+<titleabbrev>Keyword repeat</titleabbrev>
+++++
 
-The `keyword_repeat` token filter Emits each incoming token twice once
-as keyword and once as a non-keyword to allow an unstemmed version of a
-term to be indexed side by side with the stemmed version of the term.
-Given the nature of this filter each token that isn't transformed by a
-subsequent stemmer will be indexed twice. Therefore, consider adding a
-`unique` filter with `only_on_same_position` set to `true` to drop
-unnecessary duplicates.
+Outputs each token in a stream twice:
```

Comment:
Maybe "Outputs both the stemmed and unstemmed (keyword) version of each token?"

Reply:
Updated as suggested with 181c3a580b1.